src/data/reducers/assignmentTypes.test.js

```diff
@@ -24,9 +19,8 @@ describe('assignmentTypes reducer', () => {
       ...initialState,
       startedFetching: true,
     };
-    expect(assignmentTypes(undefined, {
-      type: STARTED_FETCHING_ASSIGNMENT_TYPES,
-    })).toEqual(expected);
+    expect(assignmentTypes(undefined, actions.fetching.started()))
```

Comment:
This should test with a non-undefined state to verify that it overlays appropriately (for started and error tests)

Reply:
Also: try to stick with the syntax of:
```
expect(
  longNamedThing,
).toEqual(expected)
```
vs going back and forth.